src/main/java/ui/BoardAutoCreator.java

```diff
@@ -150,7 +152,9 @@ private void createSampleBoard() {
 
         triggerBoardSaveEventSequence(SAMPLE_BOARD);
 
-        DialogMessage.showInformationDialog("Auto-create Board - " + SAMPLE_BOARD, SAMPLE_BOARD_DIALOG);
+        if (isDialogShown){
+            DialogMessage.showInformationDialog("Auto-create Board - " + SAMPLE_BOARD, SAMPLE_BOARD_DIALOG);
+        }
```

Comment:
Got it. Showing the dialog is an additional step that you do at the end, so an alternative implementation could be like this:
```
void createSampleBoard(){...}

void createSampleBoardAndInformUser(){
  createSampleBoard();
  //show dialog
}
```
Using boolens to control behaviors is not wrong, but not pretty either.
It's ok to keep the current one.